lib/std/io_uring.zig

```diff
@@ -740,17 +740,10 @@ test "queue_readv" {
 
     const fd = try os.openZ("/dev/zero", os.O_RDONLY | os.O_CLOEXEC, 0);
     defer os.close(fd);
-    
-    var registered_fds = [_]i32{-1} ** 10;
```

Comment:
The Zig build server is currently running Ubuntu 18.04.5 LTS that ships with Linux Kernel 5.4.

Linux Kernel 5.4 supports `IORING_REGISTER_FILES` but does not support sparse sets (i.e. an fd of -1) in the fd array.

The result was our test returning an EBADF for /dev/zero.

Compare:
https://github.com/torvalds/linux/blob/v5.4/fs/io_uring.c#L3119-L3124 vs
https://github.com/torvalds/linux/blob/v5.8/fs/io_uring.c#L6687-L6691